tests/db/helpers/common.js

```diff
@@ -20,6 +20,40 @@ const common = require('../../../db/helpers/common');
 
 describe('tests/db/helpers/common.js >', () => {
   after(u.forceDelete);
+
+  describe('tests/db/helpers/common.js, sortArrayAccordingToAnotherArray >', () => {
```

Comment:
I think we want the file location only in the top most describe.

Reply:
ok taking it out